src/Suave.Swagger/Swagger.fs

```diff
@@ -335,6 +335,7 @@ module Swagger =
               typeof<byte list>, ("string", "binary")
               typeof<byte seq>, ("string", "binary")
               typeof<byte>, ("string", "byte")
+              typeof<Guid>, ("string", "string")
```

Comment:
As mentioned in the comments in the swashbuckle issue, "uuid" is not a supported value for format in swagger (http://swagger.io/specification/) and it is not interpreted by the swagger ui.